sources/packages/backend/libs/services/src/disbursement-schedule/disbursement-schedule-service.ts

```diff
@@ -73,14 +81,26 @@ const GRANTS_TYPES = [
  * - Any deviation from the above statements is considered an edge case and must be adjusted and adjusted manually by the Ministry.
  */
 @Injectable()
-export class DisbursementScheduleService extends RecordDataModelService<DisbursementSchedule> {
+export class DisbursementSchedulerService extends RecordDataModelService<DisbursementSchedule> {
```

Comment:
This seems to be incorrectly named. I would say that it should be `DisbursementScheduleService` unless I am missing something.